scripts/loki_transform.py

```diff
@@ -193,260 +133,29 @@ def convert(
     # If requested, apply a custom pipeline from the scheduler config
     # Note that this new entry point will bypass all other default
     # behaviour and exit immediately after.
-    if mode in config.pipelines:
-        info(f'[Loki-transform] Applying custom pipeline {mode} from config:')
-        info(str(config.pipelines[mode]))
-
-        scheduler.process(config.pipelines[mode], proc_strategy=processing_strategy)
-
-        mode = mode.replace('-', '_')  # Sanitize mode string
-
-        # Write out all modified source files into the build directory
-        file_write_trafo = scheduler.config.transformations.get('FileWriteTransformation', None)
-        if not file_write_trafo:
-            file_write_trafo = FileWriteTransformation(cuf='cuf' in mode)
-        scheduler.process(transformation=file_write_trafo, proc_strategy=processing_strategy)
-
-        if plan_file is not None:
-            scheduler.write_cmake_plan(plan_file, rootpath=root)
-
-        if callgraph:
-            scheduler.callgraph(callgraph)
-
-        return
-
-    if plan_file is not None:
-        msg = '[Loki] ERROR: Plan mode requires a pipeline definition in the config file.\n'
+    if not mode in config.pipelines:
+        msg = '[Loki] ERROR: Pipeline or transformation mode not found in config file.\n'
```

Comment:
Might be helpful to mention the specified mode here, to spot typos quickly:
```suggestion
        msg = f'[Loki] ERROR: Pipeline or transformation mode "{mode}" not found in config file.\n'
```